integration_tests/integration/probationPractitionerReferrals.spec.js

```diff
@@ -80,15 +86,15 @@ describe('Probation practitioner referrals dashboard', () => {
         {
           Referral: 'ABCABCA1',
           'Service user': 'George Michael',
-          'Service category': 'Accommodation',
+          'Service category': 'Accommodation Services - West Midlands',
```

Comment:
Could we change the key to "contract title", "intervention type" or similar?

Reply:
Good call - will do

Reply:
Maybe just "intervention"?